src/libcollections/btree/node.rs

```diff
@@ -413,6 +414,17 @@ impl<BorrowType, K, V, Type> NodeRef<BorrowType, K, V, Type> {
         let len = self.len();
         Handle::new_edge(self, len)
     }
+
+    pub fn first_kv(self) -> Handle<Self, marker::KV> {
```

Comment:
Since it may be easy to forget that not every node will have a first or last K/V pair, especially given that `first_edge` and `last_edge` always work, it'd be nice to have a comment reminding people that this method only works for nonempty nodes.